.azure-pipelines/apiscan.yml

```diff
@@ -0,0 +1,45 @@
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily run
```

Comment:
Will we merge the APIScan task in to the static code analyze pipeline https://dev.azure.com/mseng/VSIoT/_build?definitionId=11321&_a=summary?

Reply:
@LongOddCode is busy with other tasks currently, I'm creating dedicated pipeline for api scan. We can discuss further on how we manage the analyze tasks.